### PR TITLE
Updates Randr-api-routing with esLint

### PR DESCRIPTION
**Pull request by mhhutton**

### CODE REVIEW

File: client/src/components/reviews/MetaData/ProductReviewChart.jsx [hunk 1]
```diff
@@ -84,116 +89,63 @@ function valuetext(value) {
...
+      const charNumVal = revsMetaData.characteristics[key].value;
+      const charArr = [key, Math.round(Number(charNumVal)), getCharLowHighVals(key)];
+      resultArr.unshift(charArr);
+    });
+    setCharValPairs(resultArr);
+  };
+
+  useEffect(() => {
+    if (revsMetaData) {
+      if (revsMetaData.characteristics) {
```
Comment (Pix3lAssassin) [inline client/src/components/reviews/MetaData/ProductReviewChart.jsx:108]: Could make one line using &&

File: client/src/components/reviews/MetaData/RecommendPercentage.jsx [hunk 2]
```diff
@@ -30,31 +30,31 @@ const recommendPercentage = () => {
         );
       }
     }
-  }
+  };
 
   const calcRecommendPercent = function(totFalse, totTrue) {
     if (totTrue !== NaN && totFalse !== NaN && totTrue !== undefined && totFalse !== undefined) {
```
Comment (Pix3lAssassin) [inline client/src/components/reviews/MetaData/RecommendPercentage.jsx:36]: could probably clean up this line to just (totTrue && totFalse)

File: client/src/components/reviews/RatingsAndReviews.jsx [hunk 3]
```diff
@@ -1,119 +1,138 @@
...
+import ReviewDialog from './AddReviewModal/AddReviewModal.jsx';
 import MetaData from './MetaData/MetaData.jsx';
-//Context import
+// Context import
 import { AppContext, ReviewsContext } from '../../helpers/context';
-//import helper functions
-import { getTotalReviews } from './helperFunctions.jsx';
 
-const { GITHUB_API_KEY } = require('../../../../config.js');
+const { GITHUB_API_KEY } = require('../../../../config');
```
Comment (Pix3lAssassin) [inline client/src/components/reviews/RatingsAndReviews.jsx:15]: API key isn't necessary for client requests

File: client/src/components/reviews/RatingsAndReviews.jsx [hunk 4]
```diff
@@ -1,119 +1,138 @@
...
+
+            <Grid item className={classes.componentTitle}>
+              Rating and Reviews
+            </Grid>
+
+            <Grid item className={classes.totalRev}>
+              <span>Total Reviews:</span>
+              <div>
+                <TotalReviewCount />
+              </div>
```
Comment (Pix3lAssassin) [inline client/src/components/reviews/RatingsAndReviews.jsx:98]: Could just be a string template `` `Total Reviews: ${totalRevsCount}` ``

File: client/src/components/reviews/ReviewTile/ImageGallery.jsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+const ImageGallery = (props) => {
+  const { reviewResults } = useContext(ReviewsContext);
+  const [photosArr, setPhotosArr] = useState([]);
+
+  const createImageGallery = () => {
+    setPhotosArr(props.revPhotos);
+  };
+  useEffect(() => {
+    createImageGallery();
+  }, [reviewResults]);
```
Comment (Pix3lAssassin) [inline client/src/components/reviews/ReviewTile/ImageGallery.jsx:35]: is unnecessary could just use the props provided

File: client/src/components/reviews/RatingsAndReviews.jsx [hunk 6]
```diff
@@ -1,119 +1,138 @@
...
-//   if(reviews.results) {
-//   if (reviews.results.length !== 0) {
-//   console.log('this is results[0] in ranr', reviews.results[0]);
-//   }
-// }
+  // -------------CONSOLE LOGS
+  //   console.log('this is reviews in ranr',reviews);
+  //   console.log('this is results in ranr', reviews.results);
+  //   if(reviews.results) {
+  //   if (reviews.results.length !== 0) {
```
Comment (ThinkCreatively) [inline client/src/components/reviews/RatingsAndReviews.jsx:60]: For Clean up maybe take out the comments, at least for the push to master?

File: client/src/components/reviews/ReviewTile/ReviewTile.jsx [hunk 7]
```diff
@@ -1,35 +1,50 @@
...
 import ProductRecommend from './ProductRecommend.jsx';
 import TopContainer from './TopContainer.jsx';
 import ReviewBody from './ReviewBody.jsx';
+import ImageGallery from './ImageGallery.jsx';
 
 const ReviewTile = () => {
-  const { reviews } = useContext(ReviewsContext);
+  const { reviewResults } = useContext(ReviewsContext);
   const classes = useStyles();
+  // const [helpfulnessCount, setHelpfulnessCount] = useState(0);
```
Comment (ThinkCreatively) [inline client/src/components/reviews/ReviewTile/ReviewTile.jsx:16]: Potential clean up ?

**Review by Pix3lAssassin — APPROVED** [review]

**Review by ThinkCreatively — APPROVED** [review]

**Pull request merged**